### PR TITLE
update README with third-party crates

### LINKED ISSUE #641: Implement an Ics06 SoloMachine client
## Feature Summary

 Implement an Ics06 SoloMachine client.

link: https://github.com/cosmos/ibc-go/tree/main/modules/light-clients/06-solomachine

## Proposal

you can assign me the task and I will complete the requirement.

### PR DESCRIPTION
**Pull request by plafer**
Closes: #641

### CODE REVIEW

File: crates/ibc/README.md [hunk 1]
```diff
@@ -13,6 +13,12 @@ Implementation of the Inter-Blockchain Communication Protocol ([IBC]) in Rust.
 
 See documentation on [docs.rs][docs-link].
 
+## Third-party apps
```
Comment (Farhad-Shabani) [inline crates/ibc/README.md]: Doesn't the term "apps" seem a bit confusing? by implying it's only about IBC applications, when it actually includes both apps and clients.

Comment (plafer) [reply crates/ibc/README.md]: I was thinking we'd have another list for clients

Comment (Farhad-Shabani) [reply crates/ibc/README.md]: So, the Solo machine should go under the clients' list?

Comment (plafer) [reply crates/ibc/README.md]: oh boy, I meant client 😅 [2de6c97](https://github.com/cosmos/ibc-rs/pull/710/commits/2de6c97ccd9d3893807ed7d4a451e1d2ce44dae2)

**Review by Farhad-Shabani — APPROVED** [review]
:pray:

**Pull request merged**